docs/devguide/introduction.rst

```diff
@@ -749,10 +953,10 @@ MPC: The Makefile, Project, and Workspace Creator
     Sect<1.4.1>
 
 OpenDDS is itself built with MPC, so development systems that are set up to use OpenDDS already have MPC available.
-The OpenDDS configure script creates a “setenv” script with environment settings (``setenv.cmd`` on Windows; ``setenv.sh`` on Linux/macOS).
+The OpenDDS configure script creates a "setenv" script with environment settings (``setenv.cmd`` on Windows; ``setenv.sh`` on Linux/macOS).
```

Comment:
There are several monospaced words above this in the optional features that have single quotes around them, like `‘``GROUP``’`, could you remove the outer redundant quotes?